sklearn/mixture/gaussian_mixture.py

```diff
@@ -648,3 +672,22 @@ def aic(self, X):
             The greater the better.
         """
         return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
+
+    @property
+    def precisions_(self):
+        self._check_is_fitted()
+        if self.precision_type is 'full':
+            return np.array([np.dot(prec, prec.T)
+                            for prec in self._precisions])
+        elif self.precision_type is 'tied':
+            return np.dot(self._precisions, self._precisions.T)
+        else:
+            return self._precisions
+
+    @property
+    def precisions_cholesky_(self):
+        self._check_is_fitted()
+        if self.precision_type in ['full', 'tied']:
+            return self._precisions
+        else:
+            return np.sqrt(self._precisions)
```

Comment:
I have created `_precisions` because it doesn't store a cholesky decomposition for the diagonal & spherical precisions (but directly the precision).  So I can't rename it `precision_cholesky_` directly.
Nevertheless, I don't like a lot the solution I chose.
OK for the `covariance_` property.